### PR TITLE
Bump meta-raspberrypi pinned revision

### PR DESCRIPTION
**Pull request by tkfu**
This bumps to the latest morty in meta-raspberrypi, which is stable with all our builds.

### CODE REVIEW

**Comment by cajun-rat** [comment]
Since this is part of our customer-visible release, this needs a ACK from Ben

**Comment by EpicNuts** [comment]
Is there a better way to get this change than to manually change my default.xml file to the new meta-rasp and rebuild? Like, because we're using repo there's no 'branch' for the main directory, right?

**Comment by EpicNuts** [comment]
@cajun-rat @tkfu 
1. I changed the line in my local default.xml
```
        <!-- BSP layers -->
	<project path="meta-raspberrypi" name="meta-raspberrypi" remote="yocto" revision="380bf2ff445c2049bdea15f3bb36e8cb36540345" />
```
...then repo synced. 
2. Changed to my meta-raspberrypi directory -> `ben@ben-desktop:/storage/quickstart-raspberryPi/meta-raspberrypi((HEAD detached at 380bf2f))$ ` and built two raspberryPi images.
3. Successful update from one to the other and back.

question: After confirming this and merging in, will a _repo sync_ get my local back to where it should be, or do I need to replace it with the old and then _repo sync_, or _git init_ or some other step? This change being part of the .repo folder makes it a bit different, right?

**Review by EpicNuts — APPROVED** [review]
Looks Cool To Me™

**Pull request merged**
